### DISCUSSION TITLE
Codiumate pane auto opening when launching Vscode

### DISCUSSION THREAD

**Discussion by nonoash** [Codium-ai/codiumai-vscode-release#159] [Q&A]
Hello,

I'm not sure if it's intended or i have a problem, i can't find any settings to disable this behavior.
whenever i start Vscode, the Codiumate pane that i have on the right side bar, always opens by itself and i have to hide it every time.
Is that normal? i can't even find an opened issue about it

**Comment by almog-lv** [comment]
Hey @nonoash , this is not a normal behavior. Are you talking about the test panel? (attached screenshot)
Have you closed it with the X button? 

<img width="625" alt="Screenshot 2024-04-08 at 21 41 25" src="https://github.com/Codium-ai/codiumai-vscode-release/assets/42740235/84517434-ee7d-4089-91e4-5da9f421c921">

**Reply by nonoash** [reply]
No not the test panel but the Codiumate: chat panel
After investigating, it seems (at least in my case) your extension doesn't like the side bar change.
I have Copilot and Codium on the right side panel (secondary panel) rather than the left side main panel.

If i have Codium on the left side (default location), it doesn't auto open, if i put on the right side bar, the Codiumate:chat panel open itself automatically when Vscode starts up
![codiumchat](https://github.com/Codium-ai/codiumai-vscode-release/assets/116904212/30d37a86-dbbe-4517-b594-c3239234a6b6)

**Comment by nonoash** [comment]
it is fine now
